scipy/stats/_distribution_infrastructure.py

```diff
@@ -3618,57 +3618,38 @@ class or its methods for more information.
 
     Create a custom distribution with variable support.
 
-    >>> class MyGenExtreme:
+    >>> class MyUniformCube:
     ...     @property
     ...     def __make_distribution_version__(self):
     ...         return "1.16.0"
     ...
     ...     @property
     ...     def parameters(self):
-    ...         return {"c": {}, "mu": {}, "sigma": (0, np.inf)}
+    ...         return {"a": (-np.inf, np.inf), "b": (-np.inf, np.inf)}
```

Comment:
```suggestion
    ...         return {"a": (-np.inf, np.inf), "b": ('a', np.inf)}
```
and left endpoint is not inclusive.

Reply:
Nice!